cmd/pkg.go

```diff
@@ -249,7 +249,10 @@ func parseDepSpec(cmd *cobra.Command, spec string) (registry, libID, name, versi
 	if len(split) > 2 {
 		return "", "", "", "", fmt.Errorf("Symbol '@' is only allowed once, at the end of the argument of the form <registry>/<library>@<version>")
 	}
-	version = split[len(split)-1]
+	version = ""
```

Comment:
If there is no `@`, couldn't you append a `@master`? If you made that change here, you wouldn't have to change anything else.

Reply:
Nice catch -- it's fixed! Please take another look.